README.md

```diff
@@ -51,7 +51,7 @@ The orignal v1.0 dev roadmap [is here ](https://github.com/orgs/openebs/projects
 > Before installing the LVM-LocalPV driver please make sure your Kubernetes Cluster meets the following prerequisites:
 > 1. All the nodes must have LVM2 utils package installed
 > 2. All the nodes must have dm-snapshot Kernel Module loaded - (Device Mapper Snapshot)
-> 4. You have access to install RBAC components into kube-system namespace. The OpenEBS LVM driver components are installed in kube-system namespace to allow them to be flagged as system critical components.
+> 4. You have access to install RBAC components into <OPENEBS> namespace.
 <BR>
 
 > [!NOTE]
```

Comment:
@dave-brace The below contents are not supported at the moment we might need to move it under a roadmap section. The users are still needed to created the PV and VG on their own for localpv LVM.